app/components/navigation.jsx

```diff
@@ -47,17 +45,17 @@ function ResponsiveAppBar() {
   };
 
   const handleHome = () => {
-    router.push('/')
-  }
+    router.push('/');
+  };
 
   const handleCloseUserMenu = () => {
     setAnchorElUser(null);
   };
 
   const handleUserAccount = async (setting) => {
     if (setting === 'Account') {
-      router.push('../account')
-    }
+      router.push('../account');
```

Comment:
is it best practice to use relative file paths or absolute for nextjs? I don't know the answer to this btw.

Reply:
I am not aware of any recommendations on this aspect of next.js project. I think it depends more on the complexity of the project's folder structure. For smaller projects relative paths are more convenient because all files are very close to each other